cluster-operator/src/main/java/io/strimzi/operator/cluster/model/nodepools/NodePoolUtils.java

```diff
@@ -215,21 +219,23 @@ private static List<String> validateKRaftProcessRoles(List<KafkaNodePool> nodePo
 
 
     /**
-     * In KRaft mode, JBOD storage with multiple disks is currently not supported. This method validates the
-     * KafkaNodePool resources whether they contain JBOD storage with multiple disks or not.
+     * In KRaft mode, JBOD storage with multiple disks is supported from Kafka 3.7.0. But only one disk can be marked as
+     * the one used for storing the metadata.
      *
-     * @param nodePools   List of Kafka Node Pools
+     * @param nodePools         List of Kafka Node Pools
+     * @param versionChange     Describes Kafka versions used by this cluster
      *
      * @return  List with errors found during the validation
      */
-    public static List<String> validateKRaftJbodStorage(List<KafkaNodePool> nodePools)   {
+    public static List<String> validateKRaftJbodStorage(List<KafkaNodePool> nodePools, KafkaVersionChange versionChange)   {
         List<String> errors = new ArrayList<>();
 
         for (KafkaNodePool pool : nodePools)    {
             if (pool.getSpec().getStorage() != null
                     && pool.getSpec().getStorage() instanceof JbodStorage jbod) {
-                if (jbod.getVolumes().size() > 1) {
-                    errors.add("Using more than one disk in a JBOD storage is currently not supported when the UseKRaft feature gate is enabled (in KafkaNodePool " + pool.getMetadata().getName() + ")");
+                if (KafkaVersion.compareDottedVersions(versionChange.to().version(), "3.7.0") < 0 && jbod.getVolumes().size() > 1) {
+                    // When running Kafka older than 3.7.0, JBOD storage is not supported in KRaft
+                    errors.add("Using more than one disk in a JBOD storage in KRaft mode is supported only with Apache Kafka 3.7.0 and newer (in KafkaNodePool " + pool.getMetadata().getName() + ")");
```

Comment:
Is this check going to be removed when the Strimzi operator doesn't support Kafka versions minor than 3.7.0? Should we track it?

Reply:
I opened #9960 and will add a link to the code.